### DISCUSSION TITLE
V1.6.0-rc.1

### DISCUSSION THREAD

**Discussion by vnbaaij** [microsoft/fluentui-blazor#224] [Announcements]
We've just published version 1.6.0-rc.1 of the package. There is not much new functionality added to the components (see the [release page](https://github.com/microsoft/fast-blazor/releases/tag/V1.6.0RC1) and [changelog](https://github.com/microsoft/fast-blazor/compare/V1.5.3...V1.6.0RC1)) but we now have a package that supports both .NET 6 and .NET 7! In fact, the demo environment at [https://aka.ms/fluentui-blazor](https://aka.ms/fluentui-blazor) is already running on this .NET 7 RC 1 release. 

As you can see on that site (or in the screenshot below), we did make a lot of changes to the demo environment. Layout, styling and functionality have all been altered in, what we think, a good way. The site is now much more usable on tablets and mobiles (and even has a special layout when using a Surface Duo).

<img width="554" alt="image" src="https://user-images.githubusercontent.com/1761079/190387952-1d7e7bb1-81f1-47c2-bc85-dee8b689dc6c.png">

We'll keep this package and the demo site up-to-date as new .NET 7 releases are coming out.

If you happen to be on the latest and greatest of .NET, please give this package a try and create an issue if you run into any trouble. Also, we would love to hear your feedback on the new demo and the documentation it is showing.

P.S. To have the package show up in the Visual Studio:  Go to Manage NuGet Packages for Solution -> Check the 'Include prerelease' box. If you are using the dotnet cli, add the `--prerelease` option.

**Comment by JeepNL** [comment]
Awesome! A couple of weeks ago I started a new Blazor WASM project (default Bootstrap template) and I'll try to upgrade it to Blazor-Fast Components.
